### ISSUE TITLE
Linux TravisCI failure on master

### ISSUE THREAD

**Issue by jdm** [servo/gaol#44]
```
     Running `/home/travis/build/servo/gaol/target/debug/deps/file_read_all-4db1d97cb18b5209`
thread 'main' panicked at 'assertion failed: unshare(CLONE_NEWUSER | CLONE_NEWPID) == 0', platform/linux/namespace.rs:193:4
note: Run with `RUST_BACKTRACE=1` for a backtrace.
thread 'main' panicked at 'assertion failed: libc::read(pipe_fds[0], &mut grandchild_pid as *mut i32 as *mut c_void,
           mem::size_of::<pid_t>() as size_t) ==
    mem::size_of::<pid_t>() as ssize_t', platform/linux/namespace.rs:271:8
note: Run with `RUST_BACKTRACE=1` for a backtrace.
error: test failed, to rerun pass '--test file-read-all'
```

**Comment by jdm** [comment]
@pcwalton Any chance you could take a look at this?

**Comment by jdm** [comment]
@pcwalton The TravisCI job says:
```
Distributor ID:	Ubuntu
Description:	Ubuntu 14.04.5 LTS
Release:	14.04
Codename:	trusty
Runtime kernel version: 4.14.12-041412-generic
```
That's from the "Build system information" dropdown in https://travis-ci.org/servo/gaol/jobs/358137831.

**Comment by pcwalton** [comment]
I'm having trouble reproducing this on that version of Ubuntu, unfortunately…

**Comment by Eijebong** [comment]
Fixed by #50
